src/odin/run.py

```diff
@@ -47,40 +40,15 @@ def start():
             "ECS_TASK_GROUP",
         ],
     )
+    start_migrations()
 
     schedule = sched.scheduler(time.monotonic, time.sleep)
 
-    log = ProcessLog("odin_event_loop")
+    ProcessLog("odin_event_loop")
 
     # Schedule ODIN Jobs
     schedule_sigterm_check(schedule)
     schedule_cubic_archive_qlik(schedule)
     schedule_cubic_ods_fact_gen(schedule)
 
     schedule.run()
-    log.complete()
```

Comment:
Dropped because this will never actually get called as the ECS is always in the `schedule` loop.